book/src/install.md

```diff
@@ -168,16 +168,18 @@ a c++ compiler to be installed, for example `gcc-c++`.
 
 #### Linux and macOS
 
-Either set the `HELIX_RUNTIME` environment variable to point to the runtime files and add it to your `~/.bashrc` or equivalent:
+The **runtime** directory is in the root of the `repository_directory`, so either
+set the `HELIX_RUNTIME` environment variable to point to the runtime files and
+add it to your `~/.bashrc` or equivalent:
 
 ```sh
-HELIX_RUNTIME=/home/user-name/src/helix/runtime
+HELIX_RUNTIME=repository_directory/helix/runtime
 ```
 
-Or, create a symlink in `~/.config/helix` that links to the source code directory:
+Or, create a symbolic link:
 
 ```sh
-ln -s $PWD/runtime ~/.config/helix/runtime
+ln -s repository_directory/helix/runtime ~/.config/helix/runtime
```

Comment:
I believe the link here will not resolve correctly unless repository_directory is an absolute path. The PWD version should work as long as you have `cd`d into the clone directory.

```suggestion
# Make sure you are in the Helix repository clone's directory.
ln -s $PWD/helix/runtime ~/.config/helix/runtime
```